R/aaa-auto.R

```diff
@@ -970,9 +985,7 @@ edge_betweenness_subset_impl <- function(graph, eids=E(graph), directed=TRUE, so
   on.exit( .Call(R_igraph_finalizer) )
   # Function call
   res <- .Call(R_igraph_edge_betweenness_subset, graph, eids-1, directed, sources-1, targets-1, weights)
-if (igraph_opt("add.vertex.names") && is_named(graph)) {
-    names(res) <- vertex_attr(graph, "name", V(graph))
-  }
```

Comment:
Is this intentional?

Reply:
This was incorrect, so I allowed it to be removed by the auto-generation. This is trying to add vertex names to a vector that contains edge-quantities, not vertex quantities. I also removed a dependence of the result on `V(g)` in `functions-R.yaml` for the same reason, although this seemed to have no ill effect.

If you find anything else that looks suspicious, let me know!

Reply:
Also: As far as I can tell, we're not exposing this function at the moment, so this bug did not have any consequences so far.